requirements-ci.txt

```
@@ -1,3 +1,3 @@
 strawberryfields
-pennylane
+pennylane<=0.29.0
```

just to be consistent. plus 0.29.1 exists and addressed a bug, the nature of which I do not remember 😄 
```suggestion
pennylane<0.30
```